gnomad_qc/v3/subset/subset.py

```diff
@@ -181,62 +211,112 @@ def main(args):
+                "Number": "A",
+                "Description": "Alternate allele count in subset post-filtering of low-confidence genotypes (GQ < 20; DP < 10; and AB < 0.2 for het calls)",
+                },
+             "subset_AN_adj": {
+                "Number": "1",
+                "Description": "Total number of alleles in subset post-filtering of low-confidence genotypes (GQ < 20; DP < 10; and AB < 0.2 for het calls)"
+                },
+            "subset_AF_adj": {
+                "Number": "A",
+                "Description": "Alternate allele frequency in subset post-filtering of low-confidence genotypes (GQ < 20; DP < 10; and AB < 0.2 for het calls)",
+                }
+            }
+        )
     
     if args.add_gnomad_freqs:
-        from gnomad_qc.v3.resources.annotations import freq
-        freq = freq.ht()
-        mt = mt.annotate_rows(info=mt.info.annotate(gnomAD_AC_adj=freq[mt.row_key].freq[0].AC, 
-                                                    gnomAD_AN_adj=freq[mt.row_key].freq[0].AN,
-                                                    gnomAD_AF_adj=hl.float32(freq[mt.row_key].freq[0].AF),
-                                                    gnomAD_AC_raw=freq[mt.row_key].freq[1].AC, 
-                                                    gnomAD_AN_raw=freq[mt.row_key].freq[1].AN,
-                                                    gnomAD_AF_raw=hl.float32(freq[mt.row_key].freq[1].AF)))
-        header_dict['info'].update({"gnomAD_AC_adj": {"Description": "High quality alternate allele count in gnomAD post-filtering"},
-                                    "gnomAD_AN_adj": {"Description": "High quality allele number. The total number of called alleles in gnomAD post-filtering"},
-                                    "gnomAD_AF_adj": {"Description": "High quality alternate allele frequency in gnomAD post-filtering"},
-                                    "gnomAD_AC_raw": {"Description": "Raw alternate allele count in gnomAD post-filtering"}, 
-                                    "gnomAD_AN_raw": {"Description": "Raw allele number. The total number of called alleles in gnomAD post-filtering"},
-                                    "gnomAD_AF_raw": {"Description": "Raw alternate allele frequency in gnomAD post-filtering"}})
+        freq_ht = freq.ht()
+        mt = mt.annotate_rows(info=mt.info.annotate(gnomAD_AC_adj=freq_ht[mt.row_key].freq[0].AC, 
+                                                    gnomAD_AN_adj=freq_ht[mt.row_key].freq[0].AN,
+                                                    gnomAD_AF_adj=hl.float32(freq_ht[mt.row_key].freq[0].AF),
+                                                    gnomAD_AC_raw=freq_ht[mt.row_key].freq[1].AC, 
+                                                    gnomAD_AN_raw=freq_ht[mt.row_key].freq[1].AN,
+                                                    gnomAD_AF_raw=hl.float32(freq_ht[mt.row_key].freq[1].AF)))
+        header_dict['info'].update({
```

Comment:
What is the difference between these and the AC, AN, AF... in the `HEADER_DICT` above?

Reply:
Good question, I've reached out to Laurent for help answering this. I'm thinking of dropping these to avoid any confusion.

Reply:
After speaking with Laurent and investigating the `AC` and `AC_raw` in the info file compared to the frequencies file, `AC` and `AC_raw` are computed on all samples while the frequencies ACs are computed on release only. I am dropping `AC` and `AC_raw` from the info_ht before annotating.

Reply:
Ah, yes, because this is the raw table. For the sites table I think he resets these to only the release samples (and high quality unrelated). I am assuming AN and AF also should be dropped. The only question is for subsets will we want AC, AN, and AF to now be reset to the numbers in the subset rather than having a subset_AC,... Also I think we can drop the '_adj' on everything. The way it is in gnomAD and UKBB is that we specify if it is raw, but otherwise it is adj and it is described in the header what it is

Reply:
There isn't a AN or AF in the raw table. I'm not sure I understand this sentence "The only question is for subsets will we want AC, AN, and AF to now be reset to the numbers in the subset rather than having a subset_AC" Are you suggesting renaming the `subset_AC` annotation to `AC` so on for the other callstat annotations? dropping `_adj` from the annotations and updating headers makes sense to me, especially if it is consistent with gnomAD and UKBB. Thanks again!

Reply:
Sorry, reread the comment and now I understand changing the subset_AC to AC. I think we'll just have to be clear in the email but that works for me.

Reply:
Yeah I was wondering if you think we instead of `subset_AC` we name it `AC`,  `subset_AN` we name it `AN`, and `subset_AF` we name it `AF`. Since they will be a reflection of the samples in the VCF.

Reply:
This makes sense. I've updated the annotations so if you wouldnt mind one last look, I'd appreciate it